frontend/src/routes.js

```diff
@@ -6,12 +6,14 @@ import Subscriptions from './containers/Subscriptions';
 import Transactions from './containers/Transactions';
 import DonatePage from './containers/DonatePage';
 import Leaderboard from './containers/Leaderboard';
+import OnBoarding from './containers/OnBoarding';
 
 export default (
   <Route path="/">
     <Route path="/subscriptions/:token" component={Subscriptions} />
     <Route path="/subscriptions" component={Subscriptions} />
     <Route path="/leaderboard" component={Leaderboard} />
+    <Route path="/opensource/apply" component={OnBoarding} />
```

Comment:
Since we are going to have a group on /opensource, this could be confusing. Let's rename to: /github/apply?